### PR TITLE
Issue 251 align navbar

### LINKED ISSUE #251: MAKE navbar more align
## **Describe the bug**

- see the 
Home(current)
About
Team
Contact
More 
Are in the center.

## **Possible solution**

- make it on the right side of the navbar

### PR DESCRIPTION
**Pull request by RahulBansal0502**
# Description

Please include a summary of the change and which issue is fixed.
Fixes #251 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
    - After :
## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by RahulBansal0502** [comment]
Merge my PR for issue #251 if there is any query please let me know

**Review by ravikr126 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -72,9 +72,7 @@
                     <li class="nav-item">
                         <a class="nav-link  font-weight-bold nav-items" href="team.html">Team</a>
                     </li>
-                    <li class="nav-item">
-                        <a class="nav-link  font-weight-bold nav-items" href="#footer">Contact</a>
-                    </li>
+
```
Comment (harikesh409) [inline index.html:75]: Any reason to change the order ?

File: team.html [hunk 2]
```diff
@@ -47,17 +47,35 @@
...
                 <ul class="navbar-nav ml-auto">
                     <li class="nav-item active">
-                        <a class="nav-link font-weight-bold nav-items" href="index.html">Home <span
-                                class="sr-only">(current)</span></a>
+                        <a class="nav-link font-weight-bold nav-items" href="index.html">Home </a>
                     </li>
-
                     <li class="nav-item">
-                        <a class="nav-link  font-weight-bold nav-items" href="#team">Team</a>
+                        <a class="nav-link font-weight-bold nav-items" href="index.html #about">About</a>
```
Comment (harikesh409) [inline team.html:53]: It should be `index.html#about` and not `index.html #about`

**Review by harikesh409 — CHANGES_REQUESTED** [review]
image
In small screen laptops the menu is going inside the navbar. Modify the `width` property for `.navbar-nav`, if unnecessary you can remove custom width for it.

File: index.html (hunk 1 above)
Comment (RahulBansal0502) [reply index.html:75]: yes actually if we aligned in the right then the dropdown menu's navigation will no more be aligned like this:
image

File: team.html (hunk 2 above)
Comment (RahulBansal0502) [reply team.html:53]: ok I will do that

**Comment by RahulBansal0502** [comment]
@harikesh409 now you can review if there is any issue please let me know.

File: css/style.css [hunk 3]
```diff
@@ -108,8 +108,8 @@ header {
 }
 
 .navbar-nav {
-    text-align: center;
-    width: 75% !important;
+    text-align: right;
+    /*width: 58%*/ !important;
```
Comment (harikesh409) [inline css/style.css:112]: Remove the line completely, the only `!important;` is useless.

File: index.html (hunk 1 above)
Comment (harikesh409) [reply index.html:75]: Okay

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by RahulBansal0502** [comment]
resolve Please check @harikesh409

**Review by harikesh409 — APPROVED** [review]

**Comment by RahulBansal0502** [comment]
Thanks, @harikesh409 @ravikr126

**Pull request merged**